### PR TITLE
Allow only deleting msg folders without active children labels

### PR DESCRIPTION
**Pull request by norkans7**
Replace https://github.com/nyaruka/rapidpro/pull/3106 
We only delete a folder when it has no msg labels

### CODE REVIEW

File: temba/msgs/models.py [hunk 1]
```diff
@@ -1233,6 +1233,9 @@ def toggle_label(self, msgs, add):
 
         return changed
 
+    def has_active_children_labels(self):
```
Comment (rowanseymour) [inline temba/msgs/models.py:1236]: Maybe just change this to `has_child_labels`

**Review by rowanseymour — APPROVED** [review]

File: temba/msgs/models.py (hunk 1 above)
Comment (norkans7) [reply temba/msgs/models.py:1236]: We do not really delete msg label by make is_active=False, I think that was causing the bug.
To me `has_child_labels` would include all the children

Comment (rowanseymour) [reply temba/msgs/models.py:1236]: That's true but elsewhere in the codebase, we usually treat `is_active=False` as if that thing has been deleted. For example `org.has_ticketer()`, `org.has_contacts()` etc ignore ticketers and contacts which aren't active.

Comment (norkans7) [reply temba/msgs/models.py:1236]: Okay

Comment (norkans7) [reply temba/msgs/models.py]: @rowanseymour Updated

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**